packages/design-system/src/connect/components/home-screen/home-screen.tsx

```diff
@@ -0,0 +1,31 @@
+import { Icon } from "@/powerhouse";
+import { ComponentPropsWithRef, ForwardedRef, forwardRef } from "react";
+import { twMerge } from "tailwind-merge";
+import { HomeScreenItem } from "../home-screen-item";
+type InputProps = ComponentPropsWithRef<"input">;
+type HomeScreenProps = Omit<InputProps, "className"> & {
+  readonly children: React.ReactNode;
+  readonly containerClassName?: string;
+};
+export const HomeScreen = forwardRef(function HomeScreen(
+  props: HomeScreenProps,
+  ref: ForwardedRef<HTMLInputElement>,
+) {
+  const { children, containerClassName } = props;
+  return (
+    <div>
+      <div
+        className={twMerge(
+          "container flex h-screen flex-wrap justify-center gap-4 bg-[url(/home-bg.png)] bg-cover bg-center p-8 text-gray-800 placeholder:text-gray-500",
+          containerClassName,
+        )}
+      >
+        {children}
+        <HomeScreenItem
```

Comment:
Since the caller of this component will have to pass HomeScreenItems for the existing drives anyway, I'd not add this one by default.
The HomeScreen component doesn't need to know anything about drives.

Let's create a HomeScreenCreateDriveItem instead for Connect to use instead.